Fix PPC RzIL code compilation with capstone '3' and 'next'

# capstone-3
```
FAILED: librz/analysis/librz_analysis.so.0.5.0.p/arch_ppc_ppc_il.c.o 
gcc -Ilibrz/analysis/librz_analysis.so.0.5.0.p -I. -I.. -Ilibrz -I../librz -Ilibrz/include -I../librz/include -I../librz/analysis/arch/gb -Ilibrz/util/sdb/src -I../librz/util/sdb/src -Ilibrz/util/sdb -I../librz/util/sdb -I../librz/asm/arch/include -I../librz/asm/arch -I../librz/asm/arch/h8300 -I../librz/asm/arch/hexagon -I../librz/asm/arch/msp430 -I../librz/asm/arch/rsp -I../librz/asm/arch/mcore -I../librz/asm/arch/v850 -I../librz/asm/arch/propeller -I../librz/asm/arch/ebc -I../librz/asm/arch/cr16 -I../librz/asm/arch/8051 -I../librz/asm/arch/v810 -I../librz/asm/arch/or1k -I../librz/bin/format -I../librz/type/parser -I../subprojects/capstone-3.0.5/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -O3 -Wimplicit-fallthrough=3 -DRZ_PLUGIN_INCORE=1 -D_GNU_SOURCE --std=gnu99 -Werror=sizeof-pointer-memaccess -fvisibility=hidden -Wno-cpp -fPIC -MD -MQ librz/analysis/librz_analysis.so.0.5.0.p/arch_ppc_ppc_il.c.o -MF librz/analysis/librz_analysis.so.0.5.0.p/arch_ppc_ppc_il.c.o.d -o librz/analysis/librz_analysis.so.0.5.0.p/arch_ppc_ppc_il.c.o -c ../librz/analysis/arch/ppc/ppc_il.c
../librz/analysis/arch/ppc/ppc_il.c: In function ‘ppc_is_x_form’:
../librz/analysis/arch/ppc/ppc_il.c:36:7: error: ‘PPC_INS_LBZCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_LBZX’?
   36 |  case PPC_INS_LBZCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_LBZX
../librz/analysis/arch/ppc/ppc_il.c:36:7: note: each undeclared identifier is reported only once for each function it appears in
../librz/analysis/arch/ppc/ppc_il.c:41:7: error: ‘PPC_INS_LDCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_LDX’?
   41 |  case PPC_INS_LDCIX:
      |       ^~~~~~~~~~~~~
      |       PPC_INS_LDX
../librz/analysis/arch/ppc/ppc_il.c:47:7: error: ‘PPC_INS_LHZCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_LHZX’?
   47 |  case PPC_INS_LHZCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_LHZX
../librz/analysis/arch/ppc/ppc_il.c:54:7: error: ‘PPC_INS_LWZCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_LWZX’?
   54 |  case PPC_INS_LWZCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_LWZX
../librz/analysis/arch/ppc/ppc_il.c:65:7: error: ‘PPC_INS_STBCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_STWCX’?
   65 |  case PPC_INS_STBCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_STWCX
../librz/analysis/arch/ppc/ppc_il.c:66:7: error: ‘PPC_INS_STHCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_STWCX’?
   66 |  case PPC_INS_STHCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_STWCX
../librz/analysis/arch/ppc/ppc_il.c:67:7: error: ‘PPC_INS_STWCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_STWCX’?
   67 |  case PPC_INS_STWCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_STWCX
../librz/analysis/arch/ppc/ppc_il.c:68:7: error: ‘PPC_INS_STDCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_STDCX’?
   68 |  case PPC_INS_STDCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_STDCX
../librz/analysis/arch/ppc/ppc_il.c: In function ‘ppc_get_mem_acc_size’:
../librz/analysis/arch/ppc/ppc_il.c:93:7: error: ‘PPC_INS_LBZCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_LBZX’?
   93 |  case PPC_INS_LBZCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_LBZX
../librz/analysis/arch/ppc/ppc_il.c:98:7: error: ‘PPC_INS_STBCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_STWCX’?
   98 |  case PPC_INS_STBCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_STWCX
../librz/analysis/arch/ppc/ppc_il.c:109:7: error: ‘PPC_INS_LHZCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_LHZX’?
  109 |  case PPC_INS_LHZCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_LHZX
../librz/analysis/arch/ppc/ppc_il.c:115:7: error: ‘PPC_INS_STHCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_STWCX’?
  115 |  case PPC_INS_STHCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_STWCX
../librz/analysis/arch/ppc/ppc_il.c:126:7: error: ‘PPC_INS_LWZCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_LWZX’?
  126 |  case PPC_INS_LWZCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_LWZX
../librz/analysis/arch/ppc/ppc_il.c:133:7: error: ‘PPC_INS_STWCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_STWCX’?
  133 |  case PPC_INS_STWCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_STWCX
../librz/analysis/arch/ppc/ppc_il.c:143:7: error: ‘PPC_INS_LDCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_LDX’?
  143 |  case PPC_INS_LDCIX:
      |       ^~~~~~~~~~~~~
      |       PPC_INS_LDX
../librz/analysis/arch/ppc/ppc_il.c:149:7: error: ‘PPC_INS_STDCIX’ undeclared (first use in this function); did you mean ‘PPC_INS_STDCX’?
  149 |  case PPC_INS_STDCIX:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_STDCX
../librz/analysis/arch/ppc/ppc_il.c: In function ‘ppc_moves_to_spr’:
../librz/analysis/arch/ppc/ppc_il.c:325:7: error: ‘PPC_INS_MTFSFI’ undeclared (first use in this function); did you mean ‘PPC_INS_MTFSF’?
  325 |  case PPC_INS_MTFSFI:
      |       ^~~~~~~~~~~~~~
      |       PPC_INS_MTFSF
../librz/analysis/arch/ppc/ppc_il.c: In function ‘ppc_decrements_ctr’:
../librz/analysis/arch/ppc/ppc_il.c:403:7: error: ‘PPC_INS_BCT’ undeclared (first use in this function); did you mean ‘PPC_INS_BT’?
  403 |  case PPC_INS_BCT:
      |       ^~~~~~~~~~~
      |       PPC_INS_BT
```
# capstone-next
```
../librz/analysis/arch/ppc/ppc_il.c: In function ‘ppc_decrements_ctr’:
../librz/analysis/arch/ppc/ppc_il.c:403:7: error: ‘PPC_INS_BCT’ undeclared (first use in this function); did you mean ‘PPC_INS_BT’?
  403 |  case PPC_INS_BCT:
      |       ^~~~~~~~~~~
      |       PPC_INS_BT
```

@Rot127 please use `dist-` branch name when you send PR to fix this.

--- a/librz/analysis/arch/ppc/ppc_il.c
+++ b/librz/analysis/arch/ppc/ppc_il.c
@@ -400,7 +400,6 @@ RZ_IPI bool ppc_decrements_ctr(RZ_BORROW cs_insn *insn, const cs_mode mode) {
 	case PPC_INS_BDZLA:
 	case PPC_INS_BDZLR:
 	case PPC_INS_BDZLRL:
-	case PPC_INS_BCT:
 		return !(0x4 & PPC_READ_BO_FIELD); // not BO_2
 	case PPC_INS_BDNZT:
 	case PPC_INS_BDNZTL:

--- a/librz/analysis/arch/ppc/ppc_il_ops.c
+++ b/librz/analysis/arch/ppc/ppc_il_ops.c
@@ -841,7 +841,6 @@ static RzILOpEffect *move_from_to_spr_op(RZ_BORROW csh handle, RZ_BORROW cs_insn
 	case PPC_INS_MTFSB0:
 	case PPC_INS_MTFSB1:
 	case PPC_INS_MTFSF:
-	case PPC_INS_MTFSFI:
 	case PPC_INS_MFFS:
 	case PPC_INS_MFTB:
 	case PPC_INS_MFRTCU:
@@ -1371,7 +1370,6 @@ RZ_IPI RzILOpEffect *rz_ppc_cs_get_il_op(RZ_BORROW csh handle, RZ_BORROW cs_insn
 		break;
 	// These instruction are not in the ISA manual v3.1B.
 	case PPC_INS_BRINC:
-	case PPC_INS_BCT:
 	case PPC_INS_ATTN:
 		NOT_IMPLEMENTED;
 	case PPC_INS_MFCR:
@@ -1392,7 +1390,6 @@ RZ_IPI RzILOpEffect *rz_ppc_cs_get_il_op(RZ_BORROW csh handle, RZ_BORROW cs_insn
 	case PPC_INS_MTFSB0:
 	case PPC_INS_MTFSB1:
 	case PPC_INS_MTFSF:
-	case PPC_INS_MTFSFI:
 	case PPC_INS_MTLR:
 	case PPC_INS_MTMSR:
 	case PPC_INS_MTMSRD:

--- a/librz/analysis/arch/ppc/ppc_il_ops.c
+++ b/librz/analysis/arch/ppc/ppc_il_ops.c
@@ -77,10 +77,12 @@ static RzILOpEffect *load_op(RZ_BORROW csh handle, RZ_BORROW cs_insn *insn, cons
 	case PPC_INS_LWA:
 	case PPC_INS_LWAX:
 	case PPC_INS_LWAUX:
+#if CS_API_MAJOR > 3
 	case PPC_INS_LBZCIX:
 	case PPC_INS_LHZCIX:
 	case PPC_INS_LWZCIX:
 	case PPC_INS_LDCIX:
+#endif
 		base = VARG(rA);
 		if (ppc_is_x_form(id)) {
 			disp = VARG(rB);
@@ -247,10 +249,12 @@ static RzILOpEffect *store_op(RZ_BORROW csh handle, RZ_BORROW cs_insn *insn, con
 	case PPC_INS_STHUX:
 	case PPC_INS_STWUX:
 	case PPC_INS_STDUX:
+#if CS_API_MAJOR > 3
 	case PPC_INS_STBCIX:
 	case PPC_INS_STHCIX:
 	case PPC_INS_STWCIX:
 	case PPC_INS_STDCIX:
+#endif
 		base = VARG(rA);
 		if (ppc_is_x_form(id)) {
 			disp = VARG(rB);
@@ -1207,14 +1211,12 @@ RZ_IPI RzILOpEffect *rz_ppc_cs_get_il_op(RZ_BORROW csh handle, RZ_BORROW cs_insn
 	case PPC_INS_LIS:
 	case PPC_INS_LA:
 	case PPC_INS_LBZ:
-	case PPC_INS_LBZCIX:
 	case PPC_INS_LBZU:
 	case PPC_INS_LBZUX:
 	case PPC_INS_LBZX:
 	case PPC_INS_LD:
 	case PPC_INS_LDARX:
 	case PPC_INS_LDBRX:
-	case PPC_INS_LDCIX:
 	case PPC_INS_LDU:
 	case PPC_INS_LDUX:
 	case PPC_INS_LDX:
@@ -1224,7 +1226,6 @@ RZ_IPI RzILOpEffect *rz_ppc_cs_get_il_op(RZ_BORROW csh handle, RZ_BORROW cs_insn
 	case PPC_INS_LHAX:
 	case PPC_INS_LHBRX:
 	case PPC_INS_LHZ:
-	case PPC_INS_LHZCIX:
 	case PPC_INS_LHZU:
 	case PPC_INS_LHZUX:
 	case PPC_INS_LHZX:
@@ -1234,20 +1235,23 @@ RZ_IPI RzILOpEffect *rz_ppc_cs_get_il_op(RZ_BORROW csh handle, RZ_BORROW cs_insn
 	case PPC_INS_LWAX:
 	case PPC_INS_LWBRX:
 	case PPC_INS_LWZ:
-	case PPC_INS_LWZCIX:
 	case PPC_INS_LWZU:
 	case PPC_INS_LWZUX:
 	case PPC_INS_LWZX:
+#if CS_API_MAJOR > 3
+	case PPC_INS_LBZCIX:
+	case PPC_INS_LHZCIX:
+	case PPC_INS_LWZCIX:
+	case PPC_INS_LDCIX:
+#endif
 		lop = load_op(handle, insn, mode);
 		break;
 	case PPC_INS_STB:
-	case PPC_INS_STBCIX:
 	case PPC_INS_STBU:
 	case PPC_INS_STBUX:
 	case PPC_INS_STBX:
 	case PPC_INS_STD:
 	case PPC_INS_STDBRX:
-	case PPC_INS_STDCIX:
 	case PPC_INS_STDCX:
 	case PPC_INS_STDU:
 	case PPC_INS_STDUX:
@@ -1263,7 +1267,6 @@ RZ_IPI RzILOpEffect *rz_ppc_cs_get_il_op(RZ_BORROW csh handle, RZ_BORROW cs_insn
 	case PPC_INS_STFSX:
 	case PPC_INS_STH:
 	case PPC_INS_STHBRX:
-	case PPC_INS_STHCIX:
 	case PPC_INS_STHU:
 	case PPC_INS_STHUX:
 	case PPC_INS_STHX:
@@ -1276,7 +1279,6 @@ RZ_IPI RzILOpEffect *rz_ppc_cs_get_il_op(RZ_BORROW csh handle, RZ_BORROW cs_insn
 	case PPC_INS_STVXL:
 	case PPC_INS_STW:
 	case PPC_INS_STWBRX:
-	case PPC_INS_STWCIX:
 	case PPC_INS_STWCX:
 	case PPC_INS_STWU:
 	case PPC_INS_STWUX:
@@ -1285,6 +1287,12 @@ RZ_IPI RzILOpEffect *rz_ppc_cs_get_il_op(RZ_BORROW csh handle, RZ_BORROW cs_insn
 	case PPC_INS_STXVD2X:
 	case PPC_INS_STXVW4X:
 	case PPC_INS_DCBZ:
+#if CS_API_MAJOR > 3
+	case PPC_INS_STHCIX:
+	case PPC_INS_STWCIX:
+	case PPC_INS_STBCIX:
+	case PPC_INS_STDCIX:
+#endif
 		lop = store_op(handle, insn, mode);
 		break;
 	case PPC_INS_MR:

